stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -196,6 +196,8 @@ struct RowNonZero <: PivotingStrategy end
 struct RowMaximum <: PivotingStrategy end
 struct ColumnNorm <: PivotingStrategy end
 
+const DimOrInd = Union{Integer, AbstractUnitRange}
```

Comment:
Can't we just do
```jl
using Base: DimOrInd
```
?

Reply:
Yes, that's an option, but I hadn't done this as the type union is internal to `Base`. Open to changing it though, to reduce duplication